Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'jruby-openssl', :platforms => :jruby
+gem 'rubocop', '~> 0.57.2', require: false
```

Comment:
We don't require: false, eg. https://github.com/activeadmin/activeadmin/pull/4810/commits/508287fe0d225a0c7f50bd5d50641bf8a00a7c89

Reply:
I dropped the `require: false` and it still dies.